data/sheragi/archaeology missions.txt

```diff
@@ -1170,7 +1170,11 @@ mission "Sheragi Archaeology: The Box 7b"
 			`	"I see. I'll have to warn you though, following the footsteps of a doomed species can only get you so far.`
 				goto end
 			label truth
-			`	"A Sheragi black box? That's an incredible find! Be careful about the information it contains, though. They are a doomed species. Don't you humans have a legend about a cursed box? What was it... Pandora's Box?`
+			`	You produce an image of the cube that you found on your holopad. The curator stares at it, and then slowly turns to you with pity in his eyes.`
+			`	"This box is red," he says gently. "I read a thesis once on the prevalence of color blindness in humans, but-"`
+			`	"No, wait!" you cut him off. "That's a human idiom. 'Black box' is what we use to refer to a device designed to collect and preserve flight data in the event of a catastrophic failure."`
```

Comment:
This seems out of character for how we normally have the player character interact with other characters when not given a choice, particularly with cutting the curator off in the middle of his sentence. I think it'd make more sense as a realization that the curator comes to himself, stopping himself in his own tracks, rather than being something that the player interjects with.

Reply:
I hate to disagree with you – honestly, I do – but I think that would destroy the humor of the situation, if the curator stopped himself mid-sentence. 

How about:

`You interrupt the curator and explain that 'black box', in human terminology, refers to a device used to collect and preserve flight data in the event of a catastrophic failure.`